dev.exs

```diff
@@ -87,12 +97,21 @@ defmodule DemoWeb.Router do
   end
 end
 
+defmodule DemoWeb.ProxyEndpoint do
+  use Beacon.ProxyEndpoint,
+    otp_app: :beacon,
+    session_options: [store: :cookie, key: "_beacon_dev_key", signing_salt: "pMQYsz0UKEnwxJnQrVwovkBAKvU3MiuL"],
+    fallback: DemoWeb.Endpoint
+end
+
 defmodule DemoWeb.Endpoint do
   use Phoenix.Endpoint, otp_app: :beacon
 
   @session_options [store: :cookie, key: "_beacon_dev_key", signing_salt: "pMQYsz0UKEnwxJnQrVwovkBAKvU3MiuL"]
 
-  socket "/live", Phoenix.LiveView.Socket, websocket: [connect_info: [session: @session_options]]
```

Comment:
with this removed from the existing Endpoint, will this break pages outside of beacon?  can the ProxyEndpoint work for non-beacon pages?

Reply:
This endpoint is supposed to be one of the site endpoints, probably should rename it to `DevEndooint` and create a new `DyEndpoint`. No need to have an "app endooint" in dev.exs tho because we  wouldn't use it for anything.